docs/DART_LAB/matlab/private/update_inflate.m

```diff
@@ -23,26 +29,62 @@
 
 % d (distance between obs and ens) is drawn from Gaussian with 0 mean and
 % variance: E(d^2) = \lambda^o*\sigma_p^2 + \sigma_o^2
-theta_bar_2  = ( 1 + gamma_corr * (sqrt(lambda_mean) - 1) )^2 * sigma_p_2 + sigma_o_2;
+if strcmp(flavor, 'Gaussian')
+    theta_bar_2  = ( 1 + gamma_corr * (sqrt(lambda_mean) - 1) )^2 * sigma_p_2 + sigma_o_2;
+    
+elseif strcmp(flavor, 'I-Gamma')
+    fac1 = (1 + gamma_corr * (sqrt(lambda_mean) - 1))^2;
+    fac2 = -1 / ens_size;
+    
+    if fac1 < abs(fac2), fac2 = 0; end
+    
+    theta_bar_2 = (fac1+fac2) * sigma_p_2 + sigma_o_2;
+end
+
 theta_bar    = sqrt(theta_bar_2);
 u_bar        = 1 / (sqrt(2 * pi) * theta_bar);
 like_exp_bar = - 0.5 * dist_2 / theta_bar_2;
```

Comment:
This line does not match what I think it should in adaptive_inflate_mod:linear_bayes()
```
! Compute exponent of likelihood at lambda_bar
like_exp_bar = dist_2 / (-2.0_r8 * theta_bar_2)
```

Reply:
I'm not sure I follow here. Isn't `- 0.5 * dist_2 / theta_bar_2` equivalent to `like_exp_bar = dist_2 / (-2.0_r8 * theta_bar_2)` ??